mappings/net/minecraft/datafixer/TypeReferences.mapping

```diff
@@ -1,26 +1,52 @@
 CLASS net/minecraft/class_1208 net/minecraft/datafixer/TypeReferences
+	COMMENT Represents all the type references Minecraft's datafixer can fix.
 	FIELD field_19224 POI_CHUNK Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to the point of interest data in a chunk.
 	FIELD field_23067 CHUNK_GENERATOR_SETTINGS Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to chunk generator settings.
 	FIELD field_5710 LEVEL Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5711 RECIPE Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5712 ITEM_STACK Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to an item stack.
 	FIELD field_5713 ITEM_NAME Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to an item's name.
+		COMMENT
+		COMMENT <p>The name refers to the identifier this item is registered to.
 	FIELD field_5714 TEAM Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5715 PLAYER Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to a player.
 	FIELD field_5716 STRUCTURE Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5717 OPTIONS Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to client game options.
 	FIELD field_5718 UNTAGGED_SPAWNER Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5719 ENTITY_NAME Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to a entity's name.
+		COMMENT
+		COMMENT <p>The name refers to the identifier this entity is registered to.
 	FIELD field_5720 BLOCK_STATE Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to a block state.
 	FIELD field_5721 OBJECTIVE Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5722 HOTBAR Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to the saved creative hotbars.
+		COMMENT
+		COMMENT <p>This type reference is only used on the client.
 	FIELD field_5723 ENTITY_TREE Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to an entity tree.
+		COMMENT
+		COMMENT <p>An entity tree contains the passengers of an entity and their passengers.
 	FIELD field_5724 STRUCTURE_FEATURE Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5725 ADVANCEMENTS Lcom/mojang/datafixers/DSL$TypeReference;
 	FIELD field_5726 CHUNK Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to a chunk.
 	FIELD field_5727 BLOCK_ENTITY Lcom/mojang/datafixers/DSL$TypeReference;
+		COMMENT A type reference which refers to a block entity.
```

Comment:
Is this a block entity or a block entity type?

Reply:
This refers to the block entity itself.